### PR TITLE
DOCS-308: Deploy and Manage Tenants

### PR DESCRIPTION
**Pull request by ravindk89**
This commit covers the content for:

1) Deploying a MinIO Tenant
2) Managing a MinIO Tenant
3) Updates to kubectl minio related to https://github.com/minio/operator/pull/376

### CODE REVIEW

**Comment by ravindk89** [comment]
TBD is updating the operator reference for the considerable TLS-related changes, but I'll get to that in another PR/Issue

File: source/reference/minio-kubectl-plugin.rst [hunk 1]
```diff
@@ -331,33 +359,43 @@ Expand a MinIO Tenant
 .. mc-cmd:: tenant expand
    :fullpath:
 
-   Adds a new zone to an existing MinIO Tenant.
+   Extends the total capacity of a MinIO Tenant by adding a new Pool. A Pool
+   consists of a set of MinIO Pods and their associated storage resources.
 
-   The command creates the new zone using the 
-   :minio-git:`latest release <minio/minio/releases/latest>` of ``minio``:
+   :mc-cmd:`~kubectl minio tenant expand` always uses the latest stable
```
Comment (ravindk89) [inline source/reference/minio-kubectl-plugin.rst:365]: @nitisht @Alevsk I have a feeling this isn't true.

File: source/tutorials/manage-minio-tenant.rst [hunk 2]
```diff
@@ -17,14 +17,245 @@ MinIO Tenant.
 Expand a MinIO Tenant
 ---------------------
 
-This procedure documents expanding a MinIO Tenant.
+This procedure documents expanding a MinIO Tenant by adding a new 
+Pool of MinIO Pods and storage resources.
+
```
Comment (ravindk89) [inline source/tutorials/manage-minio-tenant.rst:22]: once the Operator Console is ready for prime-time, this page will *also* include a procedure specific to using that tool vs kubectl minio.

File: source/reference/minio-kubectl-plugin.rst (hunk 1 above)
Comment (nitisht) [reply source/reference/minio-kubectl-plugin.rst:365]: `expand` command doesn't change the image. It will keep running the current image.

**Review by nitisht — APPROVED** [review]
LTGM overall

File: source/reference/minio-kubectl-plugin.rst (hunk 1 above)
Comment (ravindk89) [reply source/reference/minio-kubectl-plugin.rst]: Done,thanks.

**Review by Alevsk — APPROVED** [review]

**Pull request merged**